README.md

```diff
@@ -45,21 +45,16 @@ Production credentials: In production you'll need to add production
 credentials to database.yml
 
 ### Run
+Start the solr and rails server within a tab utilizing lando (For the first run and after `lando destroy` you must run the index task, otherwise it is optional)
 ```bash
-rake server
+lando start
+rake pulsearch:solr:index
+rails s
 ```
```

Comment:
I think it would be better to break down the steps: 
1. Start solr: lando start
2. Start the rails server:  rails s
3. Index the fixtures: rake pulsearch:solr:index

Reply:
@christinach I have updated the PR